Library/Formula/capstone.rb

```diff
@@ -2,8 +2,8 @@
 
 class Capstone < Formula
   homepage "http://capstone-engine.org"
-  url "http://capstone-engine.org/download/3.0/capstone-3.0.tgz"
-  sha1 "26e591b8323ed3f6e92637d7ee953cb505687efa"
+  url "http://capstone-engine.org/download/3.0.1/capstone-3.0.1.tgz"
+  sha1 "7f206c853c6b8d05de22e919c2455ab661654093"
   revision 1
```

Comment:
meaning that line can be deleted?

please feel free to do that, or do i need to submit a new PR?

thanks.


On Wed, Feb 4, 2015 at 7:58 AM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/capstone.rb
> <https://github.com/Homebrew/homebrew/pull/36513#discussion_r24052231>:
>
> > @@ -2,8 +2,8 @@
> >
> >  class Capstone < Formula
> >    homepage "http://capstone-engine.org"
> > -  url "http://capstone-engine.org/download/3.0/capstone-3.0.tgz"
> > -  sha1 "26e591b8323ed3f6e92637d7ee953cb505687efa"
> > +  url "http://capstone-engine.org/download/3.0.1/capstone-3.0.1.tgz"
> > +  sha1 "7f206c853c6b8d05de22e919c2455ab661654093"
> >    revision 1
>
> Can kill the revision line with the new version :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/36513/files#r24052231>.
>